packages/common/src/api/tan-query/events/utils.ts

```diff
@@ -11,3 +20,16 @@ export const getEventListQueryKey = ({ pageSize }: { pageSize?: number }) => [
   QUERY_KEYS.events,
   { pageSize }
 ]
+
+export const getEventsByEntityIdQueryKey = (
+  entityId: ID | null | undefined,
+  options?: EventsByEntityIdOptions
+) =>
+  [
+    QUERY_KEYS.eventsByEntityId,
```

Comment:
ah interesting. im not the tan-query expert, but does it make sense to have all of the events in one query key and use ids or is there a benefit to having a separate key?

Reply:
i think here we're key-ing by the entity ID, whereas in the other hook we're key-ing by the event ID itself.

probably should like, put the event in both places ideally?

i also don't think we're going to use the fetch event by event ID in the current designs? not sure